tests/acceptance/test_issue_details.py

```diff
@@ -54,6 +54,21 @@ def test_python_event(self):
         self.visit_issue(event.group.id)
         self.browser.snapshot('issue details python')
 
+    def test_python_rawbody_event(self):
+        event = self.create_sample_event(
+            platform='python-rawbody',
+        )
+        self.visit_issue(event.group.id)
+        self.browser.move_to('.request pre span')
+        self.browser.snapshot('issue details python raw body')
+
+    def test_python_formdata_event(self):
+        event = self.create_sample_event(
+            platform='python-formdata',
+        )
+        self.visit_issue(event.group.id)
+        self.browser.snapshot('issue details python formdata')
```

Comment:
Did you want the tooltip to display as well? You should be able to trigger a hover with [selenium actionchain](https://selenium-python.readthedocs.io/api.html?highlight=move_to_element#selenium.webdriver.common.action_chains.ActionChains.move_to_element).

Reply:
I did this for the one snapshot where I expect a tooltip. This one doesn't yet have a tooltip because passing annotations through is too complicated for now.